cns/multitenantcontroller/multitenantoperator/multitenantcrdreconciler.go

```diff
@@ -127,6 +127,10 @@ func (r *multiTenantCrdReconciler) Reconcile(ctx context.Context, request reconc
 			GatewayIPAddress: nc.Status.Gateway,
 		},
 		PrimaryInterfaceIdentifier: nc.Status.PrimaryInterfaceIdentifier,
+		MultiTenancyInfo: cns.MultiTenancyInfo{
+			EncapType: nc.Status.MultiTenantInfo.EncapType,
+			ID:        int(nc.Status.MultiTenantInfo.ID),
```

Comment:
since MultiTenantInfo.ID is required for CNS, could we check its value in L109 and report an error if it is not set?